src/views/lighttable.c

```diff
@@ -598,7 +598,7 @@ void leave(dt_view_t *self)
 {
   // we remove the thumbtable from main view
```

Comment:
This comment should follow the dt_thumbtable_set_parent() call.

Reply:
done